src/app/modules/profile/profile-resolver.service.ts

```diff
@@ -10,11 +11,11 @@ export class ProfileResolver implements Resolve<Profile> {
 
   resolve(): Promise<any>{
     let profile: Profile[]
-    return new Promise<any>(resolve => setTimeout(resolve, 1000))
-    .then(() => {
-      profile = this.profileSVC.profile;
+    return Promise.resolve(this.profileSVC.getProfile())
```

Comment:
Do swe need here Promise.resolve ? Try to return this.profileSVC.getProfile().then(...)